spec/lexers/hocon_spec.rb

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*- #
+
```

Comment:
Can you add '# frozen_string_literal: true'

Reply:
Fixed.